lib/core/api.js

```diff
@@ -166,8 +166,21 @@ api.user.updateProfile = function(userId, updates, cb){
     if(err){
       return cb(err);
     }
+    var currentEmail = _.get(profile, 'emails[0]');
     var newProfile = actionUtils.mergeProfileUpdates(profile, updates);
-    return api.user.addProfile(userId, newProfile, cb);
+    var emails = _.get(updates, 'emails');
+    if (_.isArray(emails) && emails.length === 1 && emails[0] !== currentEmail){
+      return async.series([
+        function(callback){return tidepool.updateCustodialUser({username: emails[0], emails: emails}, userId, callback);},
+        function(callback){return tidepool.addOrUpdateProfile(userId, newProfile, callback);},
+        function(callback){return tidepool.signupStart(userId, callback);}
+      ], function(err, results){
+        if (err){ cb(err); }
```

Comment:
looking at the rest of the code this style of single line statements doesn't match and should be updated to reflect the existing code I feel